pkgs/tools/wayland/wl-gammactl/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub
+, meson, pkg-config, ninja, git
+, wayland, wayland-scanner, wlroots, wayland-protocols, gtk3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wl-gammactl";
+  version = "master-20210913";
+
+  src = fetchFromGitHub {
+    owner = "mischw";
+    repo = pname;
+    rev = "e2385950d97a3baf1b6e2f064dd419ccec179586";
+    sha256 = "4ve38pMGKDDC4oZZERyPdxVJX4RgHwYqX6IUQQ2GpGg=";
+    fetchSubmodules = true;
```

Comment:
It seems this is only necessary because upstream has opted to vendor in `wlr-protocols`. I don't think that's needed, you can just take them from wlroots at build-time?

Reply:
Thanks @lovesegfault, I made the changes you suggested.